src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -284,10 +284,9 @@ public sealed class PythonAnalyzer : IPythonAnalyzer, IDisposable {
         private void LoadMissingDocuments(IPythonInterpreter interpreter, ImmutableArray<AnalysisModuleKey> missingKeys) {
             foreach (var (moduleName, _, isTypeshed) in missingKeys) {
                 var moduleResolution = isTypeshed ? interpreter.TypeshedResolution : interpreter.ModuleResolution;
-                moduleResolution.GetOrLoadModule(moduleName);
+                var m = moduleResolution.GetOrLoadModule(moduleName);
+                _queueTracker.Add(m.FilePath);
```

Comment:
`moduleResolution.GetOrLoadModule` doesn't invalidate module analysis. It only ensures that module (if it is available) is loaded.